app/src/main/java/com/graymandev/mvvmfoundation/view/splash/SplashActivity.kt

```diff
@@ -0,0 +1,60 @@
+package com.graymandev.mvvmfoundation.view.splash
+
+import android.content.Intent
+import android.graphics.Typeface
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.view.LayoutInflater
+import android.view.animation.Animation
+import android.widget.TextView
+import com.chetantuteja.easybinding.BindingActivity
+import com.google.android.material.animation.AnimationUtils
+import com.graymandev.mvvmfoundation.R
+import com.graymandev.mvvmfoundation.databinding.ActivitySplashBinding
+import com.graymandev.mvvmfoundation.view.intro.IntroActivity
+
+class SplashActivity : BindingActivity<ActivitySplashBinding>() {
+
+    //private var binding : ActivitySplashBinding? = null
+
+    override fun setupViewBinding(inflater: LayoutInflater): ActivitySplashBinding{
+        return ActivitySplashBinding.inflate(inflater)
+    }
+
+    override fun init(){
+        val typeFace: Typeface = Typeface.createFromAsset(assets,"cookie regular.ttf")
+
+        binding.tvAppName.typeface = typeFace
+
+        // Here we will launch the Intro Screen after the splash screen using the handler.
+        // As using handler the splash screen will disappear after what we give to the handler.
+        // Adding the handler to after the a task after some delay.
+        Handler(Looper.getMainLooper()).postDelayed(
```

Comment:
Don't use `Handler(Looper)` default constructors. We have a singleton Handler in HandlerModule.kt

```
///CharacterListFragment.kt (17:1)

@AndroidEntryPoint
class CharacterListFragment : BindingFragment<FragmentCharacterListBinding>() {

    private val rmViewModel : RickAndMortyViewModel by activityViewModels()
    private var isFirstTime = true;
    @Inject lateinit var handler : Handler
```

You need to annotate this class with `@AndroidEntryPoint` to allow dependency injection and
`@Inject` the handler to be used as a lateinit var


Example of usage:

```
///CharacterListFragment.kt (33:15)

 handler.postDelayed(1000){
                    binding.apply {
                        loader.visibility = View.GONE
                        charactersRecyclerView.visibility = View.VISIBLE
                    }
                    isFirstTime = false
                }
 ```
 
 We inject the handler to avoid creating a new handler every time we need one. We can use the same handler for all app (singleton pattern)

Reply:
Update: according to https://developer.android.com/reference/android/os/Handler the constructor Handler(Looper) is not wrong, the actual issue you should be concerned with is that it should not be instantiated but injected